RELEASE.md

```diff
@@ -13,17 +13,39 @@ BouncyCastle signer is recommended, and use environment variables `MAVEN_GPG_KEY
 to pass over the key material and the passphrase to `maven-gpg-plugin`.
 See [maven-gpg-plugin site](https://maven.apache.org/plugins/maven-gpg-plugin/usage.html) for more information.
 
-### Release steps
+## Before release
 
 **Prerequisites:**
 * deploy snapshot: `mvn deploy -P sisu-release` for testing
 * make sure source code does not have `@since TBD`; of have, search/replace it with upcoming version
 * perform the release
 
-The "usual" Maven release:
-* `mvn release:prepare`
-* `mvn release:perform`
-* project uses <https://oss.sonatype.org/> to stage (manual step: close and release staging repository)
+## Using Njord
+
+This build publishes to Maven Central using [Sonatype Central Portal](https://central.sonatype.com/) service. To accomplish this, build
+uses [Maveniverse Njord](https://github.com/maveniverse/njord) extension. Check out [Njord plugin](https://maveniverse.eu/docs/njord/) documentation as well.
+
+Follow these steps to set up yourself to publish to Central Portal:
+
+1. Using https://central.sonatype.com/account (while logged in) generate tokens for your account.
+2. Edit your settings and add following server entry:
+   ```xml
+   <server>
+      <id>sonatype-cp</id>
+      <username>{tokenusername}</username>
+      <password>{tokenpassword}</password>
+    </server>
+   ```
+   while here, another good change for settings is this:
+   ```xml
+   <pluginGroups>
+     <pluginGroup>eu.maveniverse.maven.plugins</pluginGroup>
+   </pluginGroups>
+   ```
+3. Perform release "as usual" (execute `mvn release:prepare` followed by `mvn release:perform`)
+4. If build ended OK, you will have locally staged release, use `mvn njord:list` to check store name.
+5. If needed, verify staging content using `mvn njord:list-content -Dstore=$storeName`
+6. If all okay, publish the staged store to Sonatype Central Portal `mvn njord:publish -Dstore=$storeName -Dtarget=sonatype-cp`
```

Comment:
I think the target now became obsolete. What about the store?

Reply:
Fixed